plugins/Routing.jsx

```diff
@@ -60,6 +60,8 @@ class Routing extends React.Component {
             initiallyDocked: PropTypes.bool,
             side: PropTypes.string
         }),
+        /** Automatic zoom to the extent of the location. */
```

Comment:
What about "Automatically zoom to the extent of the route"?

Reply:
Sure, no problem.